plugin.yaml

```diff
@@ -293,9 +298,17 @@ node_types:
           Authentication properties of Kubernetes Cloud providers. Optional.
           Currently supported providers: Google Cloud Platform
 
+node_types:
+
+  cloudify.kubernetes.nodes.Master:
+    derived_from: cloudify.nodes.Root
+    properties:
+     <<: *client_config
```

Comment:
We do not want to introduce a new property to this node type. If we were going to break backward compatibility, we could just get rid of this old baby.  So you will need to either repeat the section above or define the YAML mapping here instead of inside `dsl_definitions`.